### PR TITLE
add status powering-on for blade

### PR DESCRIPTION
**Pull request by xuweibj**
Normal:
1. rpower node on --> powering-on
2. rpower node off --> powering-off
3. rpower node boot --> powering-on
4. rpower node reset --> powering-on

### CODE REVIEW

File: xCAT-server/lib/xcat/plugins/kvm.pm [hunk 1]
```diff
@@ -3072,9 +3073,10 @@ sub power {
             return (1, "Unsupported power directive '$subcommand'");
         }
     }
-
-    $newnodestatus{$newstat}=[$node];
-    xCAT_monitoring::monitorctrl::setNodeStatusAttributes(\%newnodestatus, 1);
+    if ($newstat) {
+        $newnodestatus{$newstat}=[$node];
+        xCAT_monitoring::monitorctrl::setNodeStatusAttributes(\%newnodestatus, 1);
```
Comment (daniceexi) [inline xCAT-server/lib/xcat/plugins/kvm.pm]: missed ``}``? please run ``perl -c`` to check the syntax before checking in.

Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/kvm.pm]: @daniceexi i see a `}` on line 3079

Comment (daniceexi) [inline xCAT-server/lib/xcat/plugins/kvm.pm]: @whowutwut you are right

**Comment by daniceexi** [comment]
I have reviewed the change and agree to merge.

**Pull request merged**